Singularity

```diff
@@ -122,32 +102,23 @@ From: centos:7
     tar -xzf smalt-0.7.6-bin.tar.gz --no-same-owner
     ln -s /opt/smalt-0.7.6-bin/smalt_x86_64 /bin/smalt
 
+    echo ===== Installing Haploflow ===== >/dev/null
+    apt-get install -q -y libboost-all-dev build-essential sudo git ronn cmake
+    cd /opt/
+    git clone https://github.com/hzi-bifo/Haploflow
+    cd Haploflow
+    git checkout 9a5a0ff6c3a0435e723e41f98fe82ec2ad19cf50
+    yes | sh build.sh
+
     echo ===== Installing Python packages ===== >/dev/null
+    # Install dependencies for genetracks/drawsvg
+    apt-get install -q -y libcairo2-dev
     # Also trigger matplotlib to build its font cache.
-    wget -q https://bootstrap.pypa.io/get-pip.py
-    python3 get-pip.py
-    rm get-pip.py
     cd /opt
-    pip install --quiet -r /opt/micall/requirements.txt
-    ln -s /usr/local/bin/cutadapt /usr/local/bin/cutadapt-1.11
-    python3 -c 'import matplotlib; matplotlib.use("Agg"); import matplotlib.pyplot'
-
-    # Install dependencies for genetracks/drawsvg
-    yum install -q -y cairo-devel cairo cairo-tools zlib-devel
-
-    yum groupremove -q -y 'development tools'
-    yum remove -q -y epel-release wget unzip
-    yum autoremove -q -y
-    yum clean all
-
-    rm -rf /var/cache/yum
-
-    ## CAUTION! This changes the default python command to python3!
-    ## This breaks many things, including yum!
-    ## To switch back to python2, use this command:
-    # sudo alternatives --set python /usr/bin/python2
-    alternatives --install /usr/bin/python python /usr/bin/python2 50
-    alternatives --install /usr/bin/python python /usr/local/bin/python3 60
+    pip install --upgrade pip
+    pip install -r /opt/micall/requirements-basespace.txt
```

Comment:
Why do we need `requirements-basespace.txt` instead of `requirements.txt`? Do we need the requests library for something?

Reply:
This is an oversight. I must have thought that `-basespace` means smaller.